File: app/src/main/java/com/alwaystinkering/sandbot/model/web/SandBotFile.java
package com.alwaystinkering.sandbot.model.web;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SandBotFile {
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("size")
    @Expose
    private Integer size;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getSize() {
        return size;
    }

    public void setSize(Integer size) {
        this.size = size;
    }

    @Override
    public String toString() {
        return "SandBotFile{" +
                "name='" + name + '\'' +
                ", size=" + size +
                '}';
    }
}

File: app/src/main/java/com/alwaystinkering/sandbot/data/SandBotFile.kt
package com.alwaystinkering.sandbot.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SandBotFile {
    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("size")
    @Expose
    var size: Int? = null

    override fun toString(): String {
        return "SandBotFile{" +
                "name='" + name + '\''.toString() +
                ", size=" + size +
                '}'.toString()
    }
}
